src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -554,25 +554,44 @@
         <div class="measure-tooltip">
             <p><%= instructions %></p>
             <div>
-                <div class="measure-label"><%= segmentLabel %>:</div><%= segmentLength %> <%= units%>
+                <div class="measure-label"><%= segmentLabel %>:</div>
+                <% _.each(segmentLengths, function (item, index) { %>
+                    <div><%= item.length %> <%= item.uom %></div>
+                <% }); %>
             </div>
             <div>
-                <div class="measure-label"><%= lengthLabel %>:</div><%= totalLength %> <%= units%>
+                <div class="measure-label"><%= lengthLabel %>:</div>
+                <% _.each(totalLengths, function (item, index) { %>
+                    <div><%= item.length %> <%= item.uom %></div>
+                <% }); %>
             </div>
         </div>
     </script>
 
     <script type="text/template" id="template-measure-infobubble">
         <div class="measure-bubble">
-            <% if (area) { %>
+            <% if (areas) { %>
                 <p>
                     <span class="measure-label i18n" data-i18n="Area:">Area:</span>
-                    <span><%= area %> <%= areaUnits%><sup>2</sup></span>
+                    <div>
+                        <% _.each(areas, function (item, index) { %>
```

Comment:
I looked into updating the tooltip, and it looks like it involves modifying a couple CSS files, possibly in addition to a code change somewhere. The `height` rule on the tooltip itself isn't enough to fix the problem, since there are rules for the container `div` that also have to be updated. That `div` is within the dijit info window, and it's dimension appear to be specific to the element rather than derived from a specific stylesheet, so it seemed like resizing the box might involve enough untangling to be handled separately.